vm/example_containers_test.go

```diff
@@ -63,7 +63,7 @@ println(a[1])
 
 	_, err = env.Execute(script)
 	if err != nil {
-		log.Fatalf("execute error: %v\n", err)
+		log.Fatalf("Execute error: %v\n", err)
```

Comment:
I prefer leading lower case letter.

Reply:
Define error is upper case. The reason for the upper case is because it is matching the function call. This is a log message not a error message returned by a function. A function error message should be lower case.

Reply:
please look codes in go/cmd/**/*.go, all of `log.Fatal` use lower case message.

Reply:
Done